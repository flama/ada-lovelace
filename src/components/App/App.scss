#app {
  overflow-x: hidden;
  
  &.-open {
    overflow: hidden;

    #navbar > a {
      position: fixed;
      left: 30%;
      transform: translate(-20%, 0);
      bottom: 4rem;
      z-index: 999;
      text-align: center;

      font-size: 1.188rem;
      line-height: 1.625rem;
      text-transform: uppercase;
      color: rgb(153, 137, 178);

      &:hover {
        color: rgba(#fff, 0.8);
      }
    }
  }
}
