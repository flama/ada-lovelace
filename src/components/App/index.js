import React, { Component } from 'react'
import 'whatwg-fetch'
import './App.scss'
import Timeline from '../Timeline'

const wikipedia = {
  english: 'https://en.wikipedia.org/w/api.php?action=parse&page=List_of_feminists&format=json&origin=*'
}

class App extends Component {

  render() {
    let request = new Request(wikipedia.english, {
      header: new Headers({
        'Content-Type': 'application/json; charset=UTF-8',
        'Origin': 'http://adalovelace.flama.me'
      })
    })

    window.fetch(request)
    .then(response => console.log(response))
    return <Timeline />
  }
}

export default App;
