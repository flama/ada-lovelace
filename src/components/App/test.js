import React from 'react';
import ReactDOM from 'react-dom';
import Shallow from 'react-test-renderer/shallow'
const renderer = new Shallow()

import App from '../App';

describe('App', () => {
  it('renders correctly', () => {
    const tree = renderer.render(<App />)
    expect(tree).toMatchSnapshot()
  })
})
