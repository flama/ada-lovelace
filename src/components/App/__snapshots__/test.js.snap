// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  id="app"
>
  <Home />
  <GraphPage />
  <AboutPage />
  <Share />
  <Footer />
</div>
`;
